dbt_sugar/core/connectors/snowflake_connector.py

```diff
@@ -0,0 +1,56 @@
+"""
+Module Snowflake connector.
+
+Module dependent of the base connector.
+"""
+from typing import List
```

Comment:
I don't see you calling any of the `snowflake-sqlalchemy` functions is it sure it's needed?

I thought you were going to use it to generate the connection string which can be a bit more elaborate for snowflake. But if we don't need it (unless it is a requirement for the drivers --which it might be) then maybe we do without it for now as I hate how the dependencies are managed by those people.

If it can be removed you can simply do `poetry remove snowflake-sqlalchemy`

Reply:
We went through this in our call we will keep it because you said it was required.